tests/test_run_report.py

```diff
@@ -109,6 +142,7 @@ def test_report_runs(mocked_today, mocked_run_success, mocked_email):
     for s in report_runs.cache['sample_data'].values():
         s['required_yield'] = 2000000000
         s['required_coverage'] = 4
+        s['run_elements'] = ['a_run', 'errored_run']
```

Comment:
Not the format you would encounter in the run elements

Reply:
I will be changing this in line with the suggestion above.